Makefile

```diff
@@ -0,0 +1,8 @@
+installtestdeps:
+	go get github.com/stretchr/testify/assert
+	go get golang.org/x/crypto/ssh/terminal
+	go get golang.org/x/sys/unix
+	go get golang.org/x/sys/windows
+
+.PHONY: \
```

Comment:
Isn't .PHONY supposed to be at the top of the file?

Reply:
No, I don't think so. See for example https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html